doc/content/devices/abp-vs-otaa/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "ABP vs OTAA"
+description: ""
+---
+
+This section can help you understand the differences between ABP and OTAA activation modes for LORaWAN end devices, and comprehend why using OTAA is recommended. 
+
+<!--more-->
+
+A `DevEUI` is a 64-bit unique ID assigned to an end device by the manufacturer. This value is linked to the hardware and it cannot be altered.
+
+Unlike `DevEUI`, which identifies an end device globally, a 32-bit `DevAddr` identifies the end device within the current network and all communication after joining the network is done with it. A `DevAddr` value consists of `NwkAddr` (end device address within network) prefixed by a `NwkID` (network identifier).
```

Comment:
I think we should mention that multiple (many even) end devices can have the same `DevAddr`.